packages/shared/src/react/hooks/usePagesOrInfinite.ts

```diff
@@ -85,6 +86,8 @@ export const usePagesOrInfinite: UsePagesOrInfinite = (params, fetcher, config,
   const enabled = config.enabled ?? true;
   const triggerInfinite = config.infinite ?? false;
   const keepPreviousData = config.keepPreviousData ?? false;
+  const __unstable__dependencyRevalidation = config.__unstable__dependencyRevalidation ?? true;
+  const __unstable__defaultRevalidateOnEvents = config.__unstable__defaultRevalidateOnEvents || [];
```

Comment:
Should this default be different from the one you pass in at `packages/shared/src/react/hooks/useOrganizationList.tsx`?

Reply:
I added this as "failsafe" values for that utility hook. I get I could change the types, but then I would need to pass the "empty" values from useOrganization hook. Not sure if we would be gaining something from that.